curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657c91ad5028770fc68d6116.md

```diff
@@ -50,6 +53,23 @@ assert.deepEqual(numbers, diceValuesArr);
 
 # --seed--
 
+## --before-user-code--
+
+```js
+const __oldRandom = Math.random;
+const __createRandom = () => {
+  let seed = 42;
+  const a = 1664525;
+  const c = 1013904223;
+  const mod = 2**32;
+  return () => {
+    seed = (a * seed + c) % mod;
+    return seed / mod;
+  };
+};
+Math.random = __createRandom();
+```
```

Comment:
Is this resetting the behavior of `Math.random()` for the entire page instead of just while tests run? That might be confusing for users if so

Reply:
Yeah, it is. Thanks for catching that. 

I've pushed an update confining the changes to specific tests.

Reply:
Dope, looks good to me. Not sure if there is a place to hold a common utility like this for the codebase?

Reply:
Maybe placing the function in here could be a good idea? I mean it does help with the curriculum 

https://github.com/freeCodeCamp/curriculum-helpers

Reply:
That's a good idea, I'll do that now :+1: